fuel_test/check_version.sh

```diff
@@ -7,10 +7,10 @@
 [ $DEBUG == "on" ] && set -x
 
 if [[ ! -d "/tmp/fuel-plugin-xenserver" ]]; then
-	git clone https://review.openstack.org/openstack/fuel-plugin-xenserver /tmp/fuel-plugin-xenserver
+	git clone https://review.openstack.org/openstack/fuel-plugin-xenserver -b ${BRANCH} /tmp/fuel-plugin-xenserver
```

Comment:
Yes, git.openstack.org is better. But review.openstack.org is functionally equal. Previously I copy the command from gerrit (the checkout link). We can have it replaced here too,